canonical-web-and-design/ubuntu.com
# Update the release charts for Kernel and OpenStack

## Done

- Update the release charts for Kernel and OpenStack
- Added a kernel version (thanks @steverydz) axis

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/about/release-cycle#ubuntu-kernel-release-cycle
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the data looks like the [sheet](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=550510389)


## Issue / Card

Fixes #8765

## Screenshots

![image](https://user-images.githubusercontent.com/441217/99980797-8c7dbd00-2da0-11eb-836e-a4fd0ada5892.png)
